src/index.js

```diff
@@ -177,6 +183,10 @@ class RecPlayer extends Component {
     }
   };
   onMouseUp = () => {
+    setTimeout(() => {
+      this._mouseDrag = false;
+      this._mouseDown = false;
+    }, 100);
```

Comment:
What is the purpose of the mouseDrag and mouseDown and what does it do for the wakeLock exactly?

Reply:
These do not have anything to do with wakeLock functionality per se, but there is a bug in the player code (you can test it on elma.online currently) that if you drag the viewport with mouse (to move around), after you release the mouse button it will also call playPause-function thus starting or stopping the playback. I'm not sure if this is some new feature of chrome that it will interpret a drag also as a click or if this issue has existed forever. Anyway since the wakeLock is coupled with playPause function this bug resulted in enabling and disabling the wakeLock in a weird way so it's fixed this  way.

Reply:
i think it has been there since click to pause was introduced

Reply:
Cool, makes sense